src/couch_replicator_scheduler.erl

```diff
@@ -41,7 +41,7 @@
 %% definitions
 -define(MAX_BACKOFF_EXPONENT, 10).
 -define(BACKOFF_INTERVAL_MICROS, 30 * 1000 * 1000).
-
+-define(HEALTH_THRESHOLD_MICROS, 2 * 60 * 1000 * 1000).
```

Comment:
It'd make sense to me that this threshold would be the same as the size of the job run time. But I don't have a strong opinion. Thoughts?